src/sap.ui.core/src/sap/ui/core/messagebundle.properties

```diff
@@ -546,6 +546,8 @@ TestRecorder.Inspect.Snippet.Dialect.Raw=raw selector
 TestRecorder.Inspect.Snippet.Dialect.OPA5=OPA5
 #XTIT: Title for the UIVeri5 dialect
 TestRecorder.Inspect.Snippet.Dialect.UIVeri5=UIVeri5
+#XTIT: Title for the WDI5 dialect
+TestRecorder.Inspect.Snippet.Dialect.WDI5=wdi5

```

Comment:
minor: 
as the remaining options are all upper case, it looks more consistent to me at least if this one is also upper-case... since all these options appear together in the dropdown for selecting a dialect... 
but it is up to you if "wdi5" is deliberately left lower case...

Reply:
actually we would prefer to keep wdi5 lowercase. If it is fine for you as well we would keep it that way :) 

Regards
Simon